.github/README.md

```diff
@@ -28,7 +28,7 @@ Dette vil automatisk laste inn nye endringer når du lagrer filer, noe som gjør
   
 
 ```docker
-docker run --name db -e MYSQL_ROOT_PASSWORD=PASSORD -p 3306:3306 -d docker.io/library/mariadb:latest
+docker run --name db -e TZ=Europe/Oslo -e PGTZ=Europe/Oslo -e MYSQL_ROOT_PASSWORD=PASSORD -p 3306:3306 -d docker.io/library/mariadb:latest
```

Comment:
Hvorfor bruker du `PGTZ`? Er ikke det bare for Postgres SQL?

Reply:
Kopiert feil. Bra spottet, takk! Fjerner det